app/controlplane/pkg/data/workflowcontract.go

```diff
@@ -372,7 +372,7 @@ func contractInOrgQuery(ctx context.Context, q *ent.OrganizationQuery, orgID uui
 
 	if !c.SkipGetReferences {
 		query = query.WithWorkflows(func(q *ent.WorkflowQuery) {
-			q.Where(workflow.DeletedAtIsNil())
+			q.Where(workflow.DeletedAtIsNil()).WithProject()
```

Comment:
This fixes `wf contract describe`

Reply:
I think we removed this because the inner query was very expensive during the attestation process. We might want to do it more surgically.